### DISCUSSION TITLE
pitch of voice

### DISCUSSION THREAD

**Discussion by Al-Fahrezi** [coqui-ai/TTS#2559] [General Q&A] [answered]
how to get the sound output to the desired pitch . for example the pitch awhen happy, disappointed, and sad. by adjusting the pitch of the voice

**Comment by kdavis-coqui** [comment, marked as answer]
Hi Muhammad,

You can adjust the pitch of an entire line by opening the Advanced Editor,
i.e. clicking on the Advanced Editor button circled in red below

[image: Screenshot 2023-04-27 at 09.58.51.png]

and using the using the pitch control knob, circled in red below, to raise
or lower the pitch of an entire line...

[image: Screenshot 2023-04-27 at 10.00.03.png]

Best,
Kelly

**Reply by Al-Fahrezi** [reply]
No. I mean how to model a sound that produces a certain emotion

**Reply by RifatUllah102** [reply]
Hi @kdavis-coqui,

I am trying to change the pitch also for my Bengali model. But in your previous reply, I can't find the screenshot. Can you please upload the screenshot again?

One more thing, How can I improve my model's sound? (changing the recipes, providing more data, or fine-tuning will work well) I am using vits with 20 hours of Bengali data without using custom phonemes.

**Reply by kdavis-coqui** [reply]
The screen shot was for Coqui Studio, not the open source offering. So, it wouldn't help with your Bengali model.

**Reply by RifatUllah102** [reply]
How can I improve my model's sound/pronunciation? (changing the recipes, providing more data, or fine-tuning will work well) I am using vits with 20 hours of Bengali data without using custom phonemes.

**Comment by kdavis-coqui** [comment]
HI Muhammad,

I'm not sure I understand what you mean.

Could you maybe rephrase what you want to do and give an example?

Best,
Kelly

**Reply by Al-Fahrezi** [reply]
how to make your own tts model whose sound output has a certain emotion. like on the website https://coqui.ai/. sorry my english is bad

**Reply by erogol** [reply]
We do not have emotion support for CoquiTTS currently. You need to play with the code to do that.
